src/util.h

```diff
@@ -69,15 +70,50 @@ inline std::string _(const char* psz)
 void SetupEnvironment();
 bool SetupNetworking();
 
+enum LogFlags : uint32_t {
+    LOG_NONE        = 0,
+    LOG_NET         = (1 <<  0),
```

Comment:
Instead of the LOG_ prefix, I'd propose to use a scoped enum. E.g. `LogFlags::NONE` `LogFlags::NET` etc.